drivers/display/display_sdl.c

```diff
@@ -433,7 +433,7 @@ static int sdl_display_clear(const struct device *dev)
 	const struct sdl_display_config *config = dev->config;
 	struct sdl_display_data *disp_data = dev->data;
 	uint8_t bgcolor = 0x00u;
-	size_t size;
+	size_t size = 0;
```

Comment:
If add return in the default case,
initialization here is not necessary.

Reply:
Got it. Zephyr prefers code that doesn't do unnecessary initialization, but this only works in the host environment, so it's fine.

Reply:
> For future proofing. I think it's a good practice to initialize local vars regardless of possible execution paths.

No that is a bad idea, it's not future proofing anything, it's inhibiting the compiler from giving you a warning if it detects a code path that results in using an uninitialized variable, which could result in an unexpected behavior and bug, for example, in this very case where not initializing the variable allowed the compiler to identify the path where the error path was not returning but rather proceeding to execute the rest of the function.